simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/component/chargingstation/ModelVariableAccessor.java

```diff
@@ -49,11 +49,7 @@ public String getVariableName() {
     }
 
     private SetVariableResult rejectVariable(Component component, Variable variable, SetVariableDatum.AttributeType attributeType, CiString.CiString1000 attributeValue) {
-        return new SetVariableResult()
-                .withComponent(component)
-                .withVariable(variable)
-                .withAttributeType(SetVariableResult.AttributeType.fromValue(attributeType.value()))
-                .withAttributeStatus(SetVariableResult.AttributeStatus.REJECTED);
+        return READ_ONLY_VALIDATOR.validate(component, variable, attributeType, attributeValue);
```

Comment:
Can we do validation only where it is needed?

Reply:
fixed here: https://github.com/evbox/station-simulator/pull/13/files